operator/cilium_node.go

```diff
@@ -262,10 +262,6 @@ func (c *ciliumNodeUpdateImplementation) Update(origNode, node *cilium_v2.Cilium
 	return nil, nil
 }
 
-func (c *ciliumNodeUpdateImplementation) Delete(nodeName string) error {
-	return ciliumK8sClient.CiliumV2().CiliumNodes().Delete(context.TODO(), nodeName, meta_v1.DeleteOptions{})
```

Comment:
How are these garbage collected?

Reply:
There has never been such logic to handle CN GC since it depended on the ownerReference from Kubernetes to delete CN automatically. However, @sayboras is actually implementing this missing functionality as we speak.